README.md

```diff
@@ -128,18 +128,22 @@ make test
 
 Compiling CX on Windows requires a recent version of Git to be installed. 
 
-Before compiling CX, install dependencies with `pacman` (download MSYS2):
+Pacman is a utility which manages software packages.   
 
-```
-pacman -Sy
+To install pacman, download [Mysys2](https://www.msys2.org) and run the installer.
+
+When the installation is complete, click ` Run MSYS2 now `.     
```

Comment:
We should add somewhere here, if user downloads MSYS2 for first time, to start with `pacman -Syu` command.
Since MSYS shell needs to be restarted after initial update.
I have tested this multiple times on my machine.

Reply:
Please write instructions here.

Reply:
```
pacman -Syu
```
This should be first command after which MSYS will download essential packages and dependencies.
Upon finish, it will ask to close shell. Then it needs to be opened again from Start Menu, by searching for MSYS2.